release_dashboard/lib/widgets/common/dialog_prompt.dart

```diff
@@ -4,37 +4,115 @@
 
 import 'package:flutter/material.dart';
 
-/// Widget that prompts an alert dialogue to the current window and forces the user to choose between two options.
+typedef DynamicFuture = Future<dynamic> Function();
+
+/// Function that prompts an alert to the current window and forces the user to choose between two options.
```

Comment:
Avoid using technical words in your docs. You could drop off the `Function that` as it's redundant with the function signature. Starting with `Prompts an alert` is more concise.